packages/grafana-ui/src/components/uPlot/hooks.ts

```diff
@@ -88,15 +88,20 @@ export const DEFAULT_PLOT_CONFIG = {
   },
   cursor: {
     focus: {
-      prox: 30,
+      prox: -1,
```

Comment:
This change breaks the tooltip in single mode as the returned focused series is always null. Why introducing it?

Reply:
ah, I added this because hovering caused a bunch execution and renderings of axis, so this was like an optimization